nextflow_schema.json

```diff
@@ -126,6 +119,25 @@
             ],
             "description": "The location of databases used by mikrokondo"
         },
+        "allele_schema_options": {
+            "title": "Allele Schema Options",
+            "type": "object",
+            "description": "Specify an allele calling schema.",
+            "default": "",
+            "properties": {
+                "force_allele_scheme": {
+                    "type": "string",
+                    "description": "Specify the path to an allele calling schema to be used. This schema will be used for allele calling of all samples."
```

Comment:
Could you include a regex `pattern` here like other path parameters and set the format to `directory-path`?

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/104/commits/0c74a1fc4dfd86f309bd2c2202662e8ec0e3cec9